source/operations/cert-manager/cert-manager-operator.rst

```diff
@@ -0,0 +1,239 @@
+
+      kubectl apply -f operator-ca-issuer.yaml
+
+3) Create TLS certificate
+-------------------------
+
+Now that the ``Issuer`` exists in the ``minio-operator`` namespace, cert-manager can add a certificate.
+
+The certificate from cert-manager must be valid for the following DNS domains:
+
+- ``sts``
+- ``sts.minio-operator.svc.``
+- ``sts.minio-operator.svc.<cluster domain>``
+
+  .. important::
+
+      Replace ``<cluster domain>`` with the actual value for your MinIO tenant.
+      ``cluster domain`` is the internal root DNS domain assigned in your Kubernetes cluster. 
+      Typically, this is ``cluster.local``, but confirm the value by checking your coredns configuration for the correct value for your Kubernetes cluster. 
+      
+      For example:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl get configmap coredns -n kube-system -o jsonpath="{.data}"
+
+      Different Kubernetes providers manage the root domain differently.
+      Check with your Kubernetes provider for more information.
+
+1. Create a ``Certificate`` for the domains mentioned above:
+
+   Create a file named ``sts-tls-certificate.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,12
+
+      # sts-tls-certificate.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: sts-certmanager-cert
+        namespace: minio-operator
+      spec:
+        dnsNames:
+          - sts
+          - sts.minio-operator.svc
+          - sts.minio-operator.svc.cluster.local # Replace cluster.local with the value for your domain.
+        secretName: sts-tls
+        issuerRef:
+          name: minio-operator-ca-issuer
+
+   .. important::
+   
+      The ``spec.secretName`` is not optional.
+   
+      The secret name **must** be ``sts-tls``.
+      Confirm this by setting ``spec.secretName: sts-tls`` as highlighted above.
+
+2. Apply the resource to the cluster:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f sts-tls-certificate.yaml
+
+This creates a secret called ``sts-tls`` in the ``minio-operator`` namespace.
+
+.. warning::
+  
+   Failing to provide the ``sts-tls`` secret containing the TLS certificate or providing an invalid ``key-value`` pair in the secret will prevent the STS service from starting.
+
+4) Install Operator with Auto TLS disabled
+------------------------------------------
+
+You can now :ref:`install the MinIO Operator <minio-operator-installation>`.
+
+When installing the Operator deployment, set the ``OPERATOR_STS_AUTO_TLS_ENABLED`` environment variable to ``off`` in the ``minio-operator`` container. 
+
+Disabling this environment variable prevents the MinIO Operator from issuing the certificates.
+Instead, Operator waits for the TLS certificate issued by cert-manager.
+
+There are several for defining an environment variable depending on how you install the Operator.
```

Comment:
Maybe this was intended to say "several methods" ? Not entirely sure.

Reply:
Something like that. I went with "various methods".